test/index_test.js

```diff
@@ -195,5 +195,69 @@ describe("Kinto", () => {
 
       expect(coll.db).to.be.an.instanceOf(MyAdapter);
     });
+
+    it("should set collection's plugins", () => {
+      const MyRemoteTransformer = class extends RemoteTransformer {};
+      const db = new Kinto();
+      const coll = db.collection("plop", {
+        remoteTransformers: [ new MyRemoteTransformer() ]
+      });
+
+      expect(coll._plugins.remoteTransformers).to.be.an.instanceOf(Array);
+      expect(coll._plugins.remoteTransformers.length).to.equal(1);
+      expect(coll._plugins.remoteTransformers[0])
+          .to.be.an.instanceOf(MyRemoteTransformer);
+    });
+
+    it("should set the plugins on existing collection instances", () => {
+      const MyRemoteTransformer = class extends RemoteTransformer {};
+      const db = new Kinto();
+      var coll1 = db.collection("plop");
+      var coll2 = db.collection("plop", {
+        remoteTransformers: [ new MyRemoteTransformer() ]
+      });
+
+      [coll1, coll2].map((coll) => {
+        expect(coll._plugins.remoteTransformers).to.be.an.instanceOf(Array);
+        expect(coll._plugins.remoteTransformers.length).to.equal(1);
+        expect(coll._plugins.remoteTransformers[0])
+            .to.be.an.instanceOf(MyRemoteTransformer);
+      });
+    });
+
+    it("should not unset the plugins on existing collection instances", () => {
+      const MyRemoteTransformer = class extends RemoteTransformer {};
+      const db = new Kinto();
+      var coll1 = db.collection("plop", {
+        remoteTransformers: [ new MyRemoteTransformer() ]
+      });
+      var coll2 = db.collection("plop");
+
+      [coll1, coll2].map((coll) => {
+        expect(coll._plugins.remoteTransformers).to.be.an.instanceOf(Array);
+        expect(coll._plugins.remoteTransformers.length).to.equal(1);
+        expect(coll._plugins.remoteTransformers[0])
+            .to.be.an.instanceOf(MyRemoteTransformer);
+      });
+    });
+
+    it("should update the collection's plugins on all instances", () => {
+      const MyRemoteTransformer = class extends RemoteTransformer {};
+      const MyOtherRemoteTransformer = class extends RemoteTransformer {};
```

Comment:
These are ducplicated in some other tests, maybe it's worth factorizing and expose them at the higher scope level.